EFAuditer/AuditDatabaseContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore;
+
+namespace EFAuditer
+{
+    public class AuditDatabaseContext : DbContext
+    {
+        public AuditDatabaseContext() {}
+        public AuditDatabaseContext(DbContextOptions<AuditDatabaseContext> options) : base(options) {}
+
+        public virtual DbSet<AuditLog> AuditLogs { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+                // TODO NTBS-218: Read in connection string from some kind of settings file
+                optionsBuilder.UseSqlServer("data source=localhost;initial catalog=ntbsAudit;trusted_connection=true");

```

Comment:
@jasiekmiko Can we discuss how best to do this? I presume we just want a settings file here, which gets written into by the main application somehow, for the different environments